test_suite/cloud/test_aws.py

```diff
@@ -356,7 +395,7 @@ def test_network_ipv6_setup(self, host):
         assert registered_ipv6 in host.interface('eth0', 'inet6').addresses(), \
             f'Expected IPv6 {registered_ipv6} is not being used by eth0 network adapter'
 
-    def test_redhat_cds_hostnames(self, host, instance_data_aws):
+    def test_redhat_cds_hostnames(self, host, instance_data_aws, rhel_only):
```

Comment:
Sorry, I was doing some tests here.

```suggestion
    def test_redhat_cds_hostnames(self, host, instance_data_aws):
```